src/applications/disability-benefits/view-payments/components/responsive-table/ResponsiveTable.jsx

```diff
@@ -55,6 +59,7 @@ class ResponsiveTable extends Component {
               className={`${borderClasses} ${extraClass}`}
               data-label={`${field.label}:`}
               key={`${rowIndex}-${field.value}`}
```

Comment:
Would it make more sense to use `field.label` here instead of value?... 2 lines now, there is a check if the value is `null`.

Reply:
Ah yeah, @Mottie is probably correct here. Several PRs ago, I added the null check 2 lines later in the event that data point is empty and we need a default display. If I was more thorough, I would have updated the `key` to use `field.label` instead since that should never be `null`.

Reply:
@Mottie Good callout, updated this